File: Office/WebContent/service/blogservice.js
app.factory('BlogService',function($http){
	var blogService={}
	var BASE_URL="http://localhost:8080/Office"
	
		blogService.list=function(){
		return	$http.get(BASE_URL + "/servlet/getblog")
	}
	
	blogService.waitlist=function(){
		return	$http.get(BASE_URL + "/servlet/waitlist")
	}
	blogService.addblog=function(blog){
		return	$http.post(BASE_URL + "/servlet/addblog",blog)
	}
	blogService.blogapp=function(id){
		return	$http.post(BASE_URL + "/servlet/blogapp/"+id)
	}
	blogService.blogrej=function(id){
		return	$http.post(BASE_URL + "/servlet/blogrej/"+id)
	}
	return blogService;
})

File: Office/src/com/office/service/BlogService.java
package com.office.service;

import java.util.List;

import com.office.model.Blog;

public interface BlogService {
	public Blog addblog(Blog blog);
	public List<Blog> list();
	public List<Blog> waitlist();
	public Blog blogapp(int blogid);
	public Blog blogrej(int blogid);
}
